docs/03-Filtering Patches.md

```diff
@@ -1,18 +1,51 @@
 ## Filtering Patches
 
-You can ignore local changes by setting `Palindrom.filterLocalChange` function. This function should accept a JSON Patch operation and either return it or return a falsy value. Obviously, returning a falsy value will result an ignored operation.
+### 1. By defining non-enumerable properties _(recommended)_:
+Assuming your Palindrom instance is instantiated and your `palindrom.obj` is ready, you can create a non-enumerable object child in `palindrom.obj` and use it as a local non-synced store for your needs.
+
+#### Example:
+
+```js
+Object.defineProperty(palindrom.obj, "myLocalNonSyncedObject", {
+    enumerable: false,
+    value: {},
+    writable: true
+});
+```
+Now `palindrom.obj.myLocalNonSyncedObject` is an object this is not synced with the server. This is ideal for keeping client-side related information (eg: div height, or something like `isModalWarningDisplayed = true`).
+
+Example:
+
+```js
+palindrom.obj.myLocalNonSyncedObject.divHeight = 50;
+```
+... is a local change.
+
+**Note:  The added property must be an object.***
```

Comment:
Really? Why?

Reply:
Consider:

`defineProperty(obj, 'prop', {value: 5})`;

since `prop in obj` returns `true`, JSONPatcherProxy will emit a `replace` patch when `prop` is modified, even though it never emitted an `add` op. However, when `prop` is an object it won't be proxified and any changes inside it will go unnoticed.

Reply:
If you add the property as:

```
Object.defineProperty(palindrom.obj, "prop", {
    enumerable: false,
    value: 5,
    writable: false,
    configurable: true
});
```
Then you can modify its value by redefining it:

```
Object.defineProperty(palindrom.obj, "prop", {
    enumerable: false,
    value: 6,
    writable: false,
    configurable: true
});
```

You just can't change the value using the assignment operator `palindrom.obj.prop = 7`